pym/bob/input.py

```diff
@@ -593,27 +593,33 @@ def status(self, workspacePath, dir, verbose=0):
 
         status = ""
         longStatus = ""
-        if self.__commit:
-            output = self.callGit(workspacePath, 'rev-parse', 'HEAD').rstrip()
-            if output != self.__commit:
-                status += "S"
-                longStatus +=  colorize("   > commitId: configured: {}  actual: {}\n".format(self.__commit, output), "33")
-        elif self.__tag:
-            output = self.callGit(workspacePath, 'describe', '--tags', '--always').rstrip()
-            if output != self.__tag:
-                status += "S"
-                longStatus += colorize("    > tag: configured: {} actual: {}\n".format(self.__tag, output), "33")
-        elif self.__branch:
-            output = self.callGit(workspacePath, 'rev-parse', '--abbrev-ref', 'HEAD').rstrip()
-            if output != self.__branch:
-                status += "S"
-                longStatus += colorize("    > branch: configured: {} actual: {}\n".format(self.__branch, output), "33")
-            else:
-                output = self.callGit(workspacePath, 'rev-list', 'origin/'+self.__branch+'..HEAD')
-                if len(output):
-                    status += "U"
-                    # do not print detailed status this point.
-                    # git log --branches --not --remotes --decorate will give the same informations.
+
+        output = self.callGit(workspacePath, 'ls-remote' ,'--get-url').rstrip()
```

Comment:
Looking at the man-page it seems that the remote name was mandatory until recently: https://git-scm.com/docs/git-ls-remote/2.6.2 vs. https://git-scm.com/docs/git-ls-remote/2.8.0 . If this is true then I guess most stable distros won't work...